hardware/Schematic-Design-Guide.md

```diff
@@ -72,16 +95,22 @@ After you get the reference design, you may have the concern why the reference i
 
   - Low Frequency Clock Sources
 
-    An external low frequency clock can be supplied from a crystal/ceramic resonator or from an external clock source. It can source a low-frequency clock from an external source such as a TCXO or VCXO.
-  ![](files/HW-Schematic-Design-Guide/LF-Connection.png)
+    An external low frequency clock can be supplied from a crystal/ceramic resonator or an external clock source. It can source a low-frequency clock from an external source such as a TCXO or VCXO.
+    <div align="center">
+     <img src="files\HW-Schematic-Design-Guide\LF-Connection.png">  
+    </div>
+  
   - High Frequency Clock Sources
 
-    The high frequency clock can be sourced from a crystal or ceramic resonator or from an external square or sine wave source. It can source a high-frequency clock from an external source such as a TCXO or VCXO.
-   ![](files/HW-Schematic-Design-Guide/HF-Connection.png) 
-For additional information on the external oscillators, refer to the application note, [AN0016.1](https://www.silabs.com/documents/public/application-notes/an0016.1-efm32-series-1-oscillator-design-considerations.pdf)
+    The high frequency clock can be sourced from a crystal or ceramic resonator or an external square or sine wave source. It can source a high-frequency clock from an external source such as a TCXO or VCXO.
+    <div align="center">
+     <img src="files\HW-Schematic-Design-Guide\HF-Connection.png">  
+    </div>
+   
+For additional information on the external oscillators, refer to the application note, [AN0016.1](https://www.Silabs.com/documents/public/application-notes/an0016.1-efm32-series-1-oscillator-design-considerations.pdf)
```

Comment:
I think here it is worthwhile to mention AN0016.1 for series 1 EFR32 and AN0016.2 for series 2 EFR32

Reply:
Added.